app/src/main/java/org/wikipedia/travel/MainPlannerFragment.java

```diff
@@ -242,10 +256,41 @@ public void failure(Throwable caught) {
         });
     }
 
+    private void saveDestination() {
+        CallbackTask.execute(() -> DestinationDbHelper.getInstance().createList(openTrip.getDestination()), new CallbackTask.DefaultCallback <Object>() {
+            @Override
+            public void success(Object o) {
+                super.success(o);
+                userDestinationList.add(openTrip);
+            }
+
+            @Override
+            public void failure(Throwable caught) {
+                super.failure(caught);
+            }
+        });
+    }
+
+    @Override
+    public void onDeleteDestination(long id) {
+        CallbackTask.execute(() -> DestinationDbHelper.getInstance().deleteList(userDestinationList.get((int) id)), new CallbackTask.DefaultCallback <Object>() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['<' is preceded with whitespace.](https://app.codacy.com/app/amawai/android-wikipedia-390/pullRequest?prid=1526985)